war/src/main/js/pages/project/builds-card.js

```diff
@@ -30,6 +30,9 @@ function load(options = {}) {
 
   // Avoid fetching if the page isn't active
   if (document.hidden) {
+    if (buildHistoryPage.dataset.pageHasUp === "false") {
```

Comment:
What the relevance of this condition?

It seems that the second problem (switching tabs) would still remain if I page through the build history first?

Is this just making this change consistent with a pre-existing but? In testing it seems I'm not getting AJAX refreshes on page 2 of the widget.